src/pages/ProfilePage/ProfileDetails.tsx

```
@@ -138,6 +220,541 @@ export default function ProfileDetails() {
+                  type="text"
+                  value={phoneNumber}
+                  onChange={(e) => setPhoneNumber(e.target.value)}
+                  placeholder="+970 59-123-4567"
+                  className="mb-4 w-full rounded-md border px-3 py-2"
+                />
+                <button
+                  onClick={handleSendCode}
+                  disabled={!phoneNumber.trim()}
+                  className="w-full rounded-md bg-blue-600 py-2 text-white disabled:bg-gray-300"
+                >
+                  Send code
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)